### PR TITLE
Update link references of ownership from nexB to aboutcode-org

### PR DESCRIPTION
**Pull request by chinyeungli**
The `aboutcode` is now under `AboutCode.org`

### CODE REVIEW

**Comment by jkowalleck** [comment]
thank you for the update

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ ABNF syntax as per [RFC5234: Augmented BNF for Syntax Specifications: ABNF](http
 
 ## Registered Top Level Namespaces
 
-Regardless of other licensing attributes in this repository or document,
```
Comment (jkowalleck) [inline README.md:69]: Please revert the change of these two trailing whitespace, they were intentional.

**Review by jkowalleck — CHANGES_REQUESTED** [review]

**Review by jkowalleck — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chinyeungli) [reply README.md]: Thanks. It's my editor that auto format the file.
